docs/source/tutorials/sklearn_svc.ipynb

```diff
@@ -0,0 +1,303 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.            data_ret = K.cast(K.power(x_qt @ K.transpose(y_qt), 2), "float32")
</pre><p>it is not the square of the inner product, it should be the square norm of the inner product $|\langle x\vert \rangle |^2$</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tencent-quantum-lab/tensorcircuit/pull/164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
fixed!